### PR TITLE
[Security]: Moving to a secure way of uploading assets

### PR DESCRIPTION
**Pull request by agnivade**
Using my private token has the flaw that anybody with write access can run a bad script in travis and use that to wreak havoc in my public repos.

Thanks to @mebeim for figuring this out !

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ function initialize {
   export TLDR_ARCHIVE="tldr.zip"
   export SITE_HOME="$HOME/site"
   export SITE_URL="github.com/tldr-pages/tldr-pages.github.io"
```
Comment (mebeim) [inline scripts/build.sh]:
```suggestion
```

This does not seem to be needed anymore.

File: scripts/build.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ function initialize {
...
+  export SITE_REPO_SLOG="tldr-pages/tldr-pages.github.io"
```
Comment (mebeim) [inline scripts/build.sh]:
```suggestion
  export SITE_REPO_SLUG="tldr-pages/tldr-pages.github.io"
```

Oh, also a typo here.

**Comment by sbrl** [comment]
@mebeim: I see! By `on the server`, do you mean Travis CI? Or tldr-bot?

**Comment by mebeim** [comment]
@sbrl Travis CI (you can see them in the settings on Travis' website). Easier with a deploy key directly on Travis than with the bot, since either way contributors have access to it.

File: scripts/build.sh (hunk 2 above)
Comment (agnivade) [reply scripts/build.sh]: Woops, thanks !

**Comment by agnivade** [comment]
Seems to be working !

https://travis-ci.org/tldr-pages/tldr/builds/488142599

```
Pages archive is created
Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.
[master 3f4b1e0] [TravisCI] uploaded assets after commits 1799a53d7876...1e28a0cbc0b6
 1 file changed, 0 insertions(+), 0 deletions(-)
Assets (pages archive, index) deployed to static site
```

Also, this was the 4000th commit to master :)

**Pull request merged**